roles/common/tasks/main.yml

```diff
@@ -18,6 +18,33 @@
   tags:
     - common
 
+- name: preserve hostname
+  sudo: yes
+  lineinfile:
+    dest: /etc/cloud/cloud.cfg
```

Comment:
Just a question here, why wouldn't we just put in a drop in in `/etc/cloud/config.d`?